deps/rabbit/src/rabbit_channel_tracking.erl

```diff
@@ -116,27 +110,14 @@ handle_cast({user_deleted, Details}) ->
     _ = timer:apply_after(?TRACKING_EXECUTION_TIMEOUT, ?MODULE,
             delete_tracked_channel_user_entry, [Username]),
     ok;
-handle_cast({node_deleted, Details}) ->
-    case rabbit_feature_flags:is_enabled(tracking_records_in_ets) of
-        true ->
-            ok;
-        false ->
-            Node = pget(node, Details),
-            rabbit_log_channel:info(
-              "Node '~ts' was removed from the cluster, deleting"
-              " its channel tracking tables...", [Node]),
-            delete_tracked_channels_table_for_node(Node),
-            delete_per_user_tracked_channels_table_for_node(Node)
-    end.
+handle_cast({node_deleted, _Details}) ->
+    ok.
 
 -spec register_tracked(rabbit_types:tracked_channel()) -> ok.
 -dialyzer([{nowarn_function, [register_tracked/1]}]).
 
 register_tracked(TrackedCh = #tracked_channel{node = Node}) when Node == node() ->
-    case rabbit_feature_flags:is_enabled(tracking_records_in_ets) of
-        true -> register_tracked_ets(TrackedCh);
-        false -> register_tracked_mnesia(TrackedCh)
-    end.
+    register_tracked_ets(TrackedCh).
```

Comment:
Let's delete `register_tracked_ets/1` and inline its function body directly into `register_tracked/1`